{"dependencies":[{"name":"C:\\Users\\Administrator\\Documents\\AI\\Tensotflow.js\\tfjs-examples-master\\tfjs-examples-master\\mnist-core\\package.json","includedInParent":true,"mtime":1540018993342},{"name":"C:\\Users\\Administrator\\Documents\\AI\\Tensotflow.js\\tfjs-examples-master\\tfjs-examples-master\\mnist-core\\.babelrc","includedInParent":true,"mtime":1539961594000},{"name":"C:\\Users\\Administrator\\Documents\\AI\\Tensotflow.js\\tfjs-examples-master\\tfjs-examples-master\\mnist-core\\node_modules\\core-js\\package.json","includedInParent":true,"mtime":1537025157511},{"name":"./_an-object","loc":{"line":2,"column":23}}],"generated":{"js":"// call something on iterator step with safe closing on error\nvar anObject = require('./_an-object');\nmodule.exports = function (iterator, fn, value, entries) {\n  try {\n    return entries ? fn(anObject(value)[0], value[1]) : fn(value);\n  // 7.4.6 IteratorClose(iterator, completion)\n  } catch (e) {\n    var ret = iterator['return'];\n    if (ret !== undefined) anObject(ret.call(iterator));\n    throw e;\n  }\n};\n","map":{"mappings":[{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":1,"column":0},"generated":{"line":1,"column":0}},{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":2,"column":0},"generated":{"line":2,"column":0}},{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":3,"column":0},"generated":{"line":3,"column":0}},{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":4,"column":0},"generated":{"line":4,"column":0}},{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":5,"column":0},"generated":{"line":5,"column":0}},{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":6,"column":0},"generated":{"line":6,"column":0}},{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":7,"column":0},"generated":{"line":7,"column":0}},{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":8,"column":0},"generated":{"line":8,"column":0}},{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":9,"column":0},"generated":{"line":9,"column":0}},{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":10,"column":0},"generated":{"line":10,"column":0}},{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":11,"column":0},"generated":{"line":11,"column":0}},{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":12,"column":0},"generated":{"line":12,"column":0}},{"source":"node_modules\\core-js\\library\\modules\\_iter-call.js","original":{"line":13,"column":0},"generated":{"line":13,"column":0}}],"sources":{"node_modules\\core-js\\library\\modules\\_iter-call.js":"// call something on iterator step with safe closing on error\nvar anObject = require('./_an-object');\nmodule.exports = function (iterator, fn, value, entries) {\n  try {\n    return entries ? fn(anObject(value)[0], value[1]) : fn(value);\n  // 7.4.6 IteratorClose(iterator, completion)\n  } catch (e) {\n    var ret = iterator['return'];\n    if (ret !== undefined) anObject(ret.call(iterator));\n    throw e;\n  }\n};\n"},"lineCount":13}},"hash":"f90ad458354c58703e6afd545ec3d8eb","cacheData":{"env":{}}}